Dockerfile

```diff
@@ -8,6 +8,11 @@ EXPOSE ${PRESTO_PORT}
 WORKDIR /app
 ENV DEBIAN_FRONTEND=noninteractive
 
+RUN git clone https://github.com/facebookresearch/sscd-copy-detection.git
+RUN cd sscd-copy-detection && python -m pip install -r ./requirements.txt --extra-index-url https://download.pytorch.org/whl/cu113
```

Comment:
Are we using the same docker file for all the models?  I think we need to have each one be able to have its own dependencies (so we don't install pytorch if not needed)

Reply:
Agreed that we want to get to separate Dockerfiles for each model. We'll make that change next year.